### DISCUSSION TITLE
Incorrect initial execution time in Nushell

### DISCUSSION THREAD

**Discussion by lewis-yeung** [JanDeDobbeleer/oh-my-posh#2704] [General]
**The initial execution time in Nushell is always 823ms** because of [this code](https://github.com/nushell/nushell/blob/a43514deb226592b664b21ccc230b3201f2aced7/crates/nu-cli/src/repl.rs#L65-L72). I asked the maintainer of Nushell, and he said it was an "Easter egg" (see [here](https://github.com/nushell/nushell/discussions/6402#discussioncomment-3466687)). Unfortunately, `let-env CMD_DURATION_MS = "0"` does not work. However, a workaround can resolve it:

```nu
let-env PROMPT_COMMAND = {
    let cmd_duration = if $env.CMD_DURATION_MS == "0823" { 0 } else { $env.CMD_DURATION_MS }
    let width = (term size -c | get columns | into string)
    ^::OMP:: print primary $"--config=($env.POSH_THEME)" --shell=nu $"--shell-version=($env.NU_VERSION)" $"--execution-time=($cmd_duration)" $"--error=($env.LAST_EXIT_CODE)" $"--terminal-width=($width)"
}
```

@JanDeDobbeleer Should we do this?

**Comment by JanDeDobbeleer** [comment]
@lewis-yeung what in the fresh hell? I suppose we indeed have to work around it. But I'd properly comment that code 😅
